static/gsApp/views/seerAutomation/onboardingV2.tsx

```diff
@@ -29,13 +38,30 @@ export default function SeerOnboardingV2() {
         )}
       />
 
-      <NoProjectMessage organization={organization}>
-        <SeerOnboardingProvider>
-          <StyledGuidedSteps>
-            <StepsManager />
-          </StyledGuidedSteps>
-        </SeerOnboardingProvider>
-      </NoProjectMessage>
+      {hasSeerProduct ? (
+        <NoProjectMessage organization={organization}>
+          <SeerOnboardingProvider>
+            <StyledGuidedSteps>
+              <StepsManager />
+            </StyledGuidedSteps>
+          </SeerOnboardingProvider>
+        </NoProjectMessage>
+      ) : isAllowedBilling ? (
+        <Alert type="info">
+          {tct(
+            'Seer is not enabled for your organization. [link:Go to your billing settings] to enable Seer.',
+            {
+              link: <Link to="/settings/billing/" />,
```

Comment:
oh one suggestion:

```suggestion
              link: <Link to="/settings/billing/overview/?product=seer" />,
```

this will automatically open the panel for seer which will show the appropriate billing CTA